config.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  path:
+    type: string
+    default: ""
+    description: >
```

Comment:
I think the [charm style guide](https://juju.is/docs/sdk/styleguide?dfghj#heading--charm-configuration-option-description) suggests the use of `|` over `>`. 
> Prefer to use the block style (using |) to keep new lines. Using > will replace new lines with spaces and make the result harder to read on [Charmhub.io](http://charmhub.io/).

Reply:
Minor comments